colour-mapping/README.rst

```diff
@@ -0,0 +1,45 @@
+Colour Mapping Demo
+===================
+A demo on using the D-Wave Ocean SDK to solve the colour mapping problem. Namely,
```

Comment:
I flinch every time I read [color mapping](https://en.wikipedia.org/wiki/Color_mapping) in place of [map coloring](https://en.wikipedia.org/wiki/Map_coloring).

But I see you consistently use *color mapping*. Am I missing something?

Reply:
I would add that we follow the American spelling color rather than the English/Canadian colour

Reply:
@randomir: Thanks for the link to color mapping. It was a fun read! I've made the change from colour mapping to map coloring.

@JoelPasvolsky: I've made the change to spell colour without the 'u'. I just want you to know that it hurts me a little to have to make that change :P

Reply:
@m3ller, as a non-American, I too feel that pain